### PR TITLE
Update cli.mdx to udpate TCP examples

### PR DESCRIPTION
**Pull request by justngrok**
Resolves ngrok-private/ngrok#33866

Updating ngrok TCP for --url flag and examples.

### CODE REVIEW

File: docs/agent/cli.mdx [hunk 1]
```diff
@@ -508,7 +508,7 @@ ngrok tcp 22
 
 ```sh
 # expose an RDP server on a specific public address that you reserved
-ngrok tcp --remote-addr=1.tcp.ngrok.io:27210 3389
+ngrok tcp --url=1.tcp.ngrok.io:27210 3389
```
Comment (nijikokun) [inline docs/agent/cli.mdx:511]: @KristopherPaulsen I remember seeing a comment about requiring scheme when doing TCP for some reason, is that something we should consider here?

Comment (KristopherPaulsen) [reply docs/agent/cli.mdx:511]: Yes! For now we still require explicitly adding the scheme for tcp (fix incoming soon™️).

eg) `--url=tcp://1.tcp.ngrok.io:27210`

Comment (justngrok) [reply docs/agent/cli.mdx:511]: Yep, that was my miss.

Comment (justngrok) [reply docs/agent/cli.mdx]: @nijikokun Fixed it (I think)

**Review by nijikokun — APPROVED** [review]

**Pull request merged**